### ISSUE TITLE
ocamlformat on MLI file of ocaml-base64 package

### ISSUE THREAD

**Issue by dinosaure** [ocaml-ppx/ocamlformat#513]
See [mirage/ocaml-base64](https://github.com/mirage/ocaml-base64), `ocamlformat` returns on `b64.mli`:

```
ocamlformat: Cannot process "b64.mli".
```

**Comment by gpetiot** [comment]
It is fixed by removing the space before the closing bracket:
```
   {e Release %%VERSION%% - %%PKG_HOMEPAGE%% }
```
We need this instead:
```
   {e Release %%VERSION%% - %%PKG_HOMEPAGE%%}
```

We are aware that the error message is really not helpful to the user (and a bit scary), that's why this option would be disabled by default in the next release.

**Comment by jberdine** [comment]
Is this space significant as far as what octavius parses it to? If not, 
is it feasible to loosen the docstring preservation check to allow such 
differences?

**Comment by gpetiot** [comment]
Fixed with #511
